<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesktopSettingDialog</class>
 <widget class="QDialog" name="DesktopSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image:url(&quot;:image/resource/image/skin_expand.png&quot;);
border:1px solid #e9e9e9;
border-radius:5px;
font-size: 14px;
margin-bottom: 5px;
}

/*QGroupBox{
margin-left: 5px;
margin-right: 5px;
}(/</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>305</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>8</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>8</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color:0;/*#8bc600;*/
font: bold 11pt;</string>
      </property>
      <property name="text">
       <string>paralle perial port map settings</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>33</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:#e9e9e9;
margin-right:6px;
margin-bottom:6px;
}
QLabel{
font-size: 14px;
}</string>
   </property>
   <widget class="QComboBox" name="comboBox_ss">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>43</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>43</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Server:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>43</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Local:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_sl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>43</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_ps">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>103</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LPT1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LPT2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LPT3</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_pl">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>103</y>
      <width>69</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>LPT1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LPT2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LPT3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>103</y>
      <width>61</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Local:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>103</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Server:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtn_ok">
    <property name="geometry">
     <rect>
      <x>238</x>
      <y>150</y>
      <width>72</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>26</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>26</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton{
color: rgb(255, 255, 255);
font: bold 12pt;
border-radius: 0px;                                        
background-color: #669900;}                                       
QPushButton:hover{                                        
background-color: #77b200;}                                       
QPushButton:pressed{                                        
background-color: #99cc00;}*/                                     </string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialPort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>75</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ParallePort:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
